### ISSUE TITLE
NVDA with the new microsoft edge chromium

### ISSUE THREAD

**Issue by bedenhofer** [bedenhofer/NVDA#1]
When you go to delete your history in microsoft  edge chromium with NVDA, and you go under manage history, and tab to the list view, and select your pages with your spacebar, and hit your applications key oand select the option that says delete, and enter on it, we should be able to hit the letter h for headings, and NVDA, should announce the word history, and when you hit the letter B, it should announce the button that says delete history, and when you space on it, it should give you a dialog box, asking you if you want to delet your history, then you should see a cancel button, as well as a delete button. This is not the case, it works with jaws, but not NVDA
